common/src/main/java/com/revenuecat/purchases/common/APIKeyValidator.kt

```diff
@@ -0,0 +1,64 @@
+package com.revenuecat.purchases.common
```

Comment:
I tend to move things into `common` whenever they are needed by other modules. In this case we could have the class in the `purchases` module and have the functions as `internal` instead of `public` and visible for testing. What do you think? I don't love the modules to be fair... this is a conversation we could have haha

Reply:
Yeah, it's only needed in the `purchases` module so it's totally possible to move it there and make it `internal`. My reasoning for putting it on `common` was that I didn't want to make `purchases` too big with a lot of internal/private code and try to keep a bit of separation between the public API and internal code using modules... That said, I don't know if that's a concern and I understand that `common` naming might indicate that it's used on multiple modules so I'm ok to move it there. Let's chat today so we can decide on the structure for this kind of code.

Reply:
Moved this to the `purchases` module